api/maven-api-core/src/main/java/org/apache/maven/api/Type.java

```diff
@@ -22,7 +22,7 @@
 import org.apache.maven.api.annotations.Immutable;
 
 /**
- * An artifact's{@code Type} represents a known kind of artifacts.
+ * A dependency's {@code Type} represents a known kind of dependencies.
```

Comment:
dependency (singular)

This comment gives an example, which is good, but it still doesn't fully explain what a dependency type *is*. see https://issues.apache.org/jira/browse/MNGSITE-521

Reply:
I've reworded again.